src/components/favorites/favorites.tsx

```diff
@@ -0,0 +1,52 @@
+import { Link } from 'react-router-dom';
+import { PlaceCardType } from '../../types/place-card';
+import PlaceCard from '../place-card/place-card';
+import { AppRoute } from '../../const';
+
+type FavoritesProps = {
+    cards: PlaceCardType[];
+}
+
+function Favorites({cards}: FavoritesProps): JSX.Element {
+  const cardNames = cards?.reduce((acc: string[], cur) => {
```

Comment:
```suggestion
  const cityNames = cards?.reduce((acc: string[], card) => {
```

https://up.htmlacademy.ru/react/13/criteries#b6

Reply:
сделать маппинг карточек с названием города зесь, чтобы не делать итерацию по cards 
 два раза